lib/services/plugins/nativescript-project-plugins-service.ts

```diff
@@ -567,8 +567,11 @@ export class NativeScriptProjectPluginsService extends PluginsServiceBase implem
 				};
 
 				if (jsonInfo.nativescript && jsonInfo.nativescript.platforms) {
-					let matchingVersion = semver.maxSatisfying(_.values<string>(jsonInfo.nativescript.platforms), `>=${this.$project.projectData.FrameworkVersion}`);
-					if (!matchingVersion) {
+					const requiredVersions = _.values<string>(jsonInfo.nativescript.platforms);
+
+					const notSupportedValues = requiredVersions.filter(ver => semver.gt(ver, this.$project.projectData.FrameworkVersion));
```

Comment:
I'd prefer the filter condition be
`semver.valid(ver) && semver.gt(ver, this.$project.projectData.FrameworkVersion)`

In order to avoid a runtime exception in the case of an invalid plugin